tcms/management/management/commands/baseurl.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = ("Sets the base URL of Kiwi TCMS instance. "
+            "If no arguments given returns current base URL."
+            )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'baseurl', nargs='?', default=None,
+            help='Base URL of Kiwi TCMS instance',
+        )
+
+    def handle(self, **kwargs):
+        siteid = settings.SITE_ID
+        site = Site.objects.get(id=siteid)
+        if not kwargs['baseurl']:
+            self.stdout.write('URL: "%s", name: "%s"' % (site.domain, site.name))
+            return
```

Comment:
We probably want a non-0 exit code here. IDK what other commands return in case of errors. If they still return 0 then leave it like that.

Reply:
What about this? How do other commands behave ?